### DISCUSSION TITLE
Oh no can't tp to lustia

### DISCUSSION THREAD

**Discussion by LeroyNyankins** [ErinaSugino/Starbound-SBR-Mod#135] [Trouble Shooting] [answered]
I just can't teleport there, don't know why and what mod can cause that, checked all compatibility notes.
[starbound.log](https://github.com/user-attachments/files/17246226/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
The mod that's causing that is SBR and the solution to fix it right now is to manually download [this](https://github.com/ErinaSugino/Starbound-SBR-Mod/blob/main/tilesets/packed/objects-special/sexscape_tileset.json) file from the repo and replace it in your mod.

**Reply by LeroyNyankins** [reply]
Yay it works now! Thanks, will wait official fix update too.
